composer.json

```diff
@@ -35,14 +35,22 @@
-      "sed -I '' \"s/{% include '/{% include '@ts_motif\\//g\" templates/*.twig templates/*/*.twig templates/*/*/*.twig",
-      "sed -I '' 's/{% extends \"/{% extends \"@ts_motif\\//g' templates/*.twig templates/*/*.twig templates/*/*/*.twig",
-      "sed -I '' \"s/{% extends '/{% extends '@ts_motif\\//g\" templates/*.twig templates/*/*.twig templates/*/*/*.twig",
-      "sed -I '' 's/featured_image: Image(308)//g' templates/*.twig templates/*/*.twig templates/*/*/*.twig",
-      "sed -I '' 's/__(//g' templates/*.twig templates/*/*.twig templates/*/*/*.twig",
-      "sed -I '' \"s/, 'ts')/|t/g\" templates/*.twig templates/*/*.twig templates/*/*/*.twig",
-      "sed -I '' \"s/,'ts')/|t/g\" templates/*.twig templates/*/*.twig templates/*/*/*.twig"
+      "sed -I '' 's/{% include \"/{% include \"@ts_motif\\//g' templates/*.twig templates/*/*.twig templates/*/*/*.twig templates/*/*/*/*.twig templates/*/*/*/*/*.twig",
+      "sed -I '' \"s/{% include '/{% include '@ts_motif\\//g\" templates/*.twig templates/*/*.twig templates/*/*/*.twig templates/*/*/*/*.twig templates/*/*/*/*/*.twig",
+      "sed -I '' 's/{% extends \"/{% extends \"@ts_motif\\//g' templates/*.twig templates/*/*.twig templates/*/*/*.twig templates/*/*/*/*.twig templates/*/*/*/*/*.twig",
+      "sed -I '' \"s/{% extends '/{% extends '@ts_motif\\//g\" templates/*.twig templates/*/*.twig templates/*/*/*.twig templates/*/*/*/*.twig templates/*/*/*/*/*.twig",
+      "sed -I '' \"s/{{ source('/{{ source('@ts_motif\\//g\"   templates/*.twig templates/*/*.twig templates/*/*/*.twig templates/*/*/*/*.twig templates/*/*/*/*/*.twig",
+      "sed -I '' \"s/{{ source(\"/{{ source(\"@ts_motif\\//g\" templates/*.twig templates/*/*.twig templates/*/*/*.twig templates/*/*/*/*.twig templates/*/*/*/*/*.twig",
+      "sed -I '' \"s/{% embed '/{% embed '@ts_motif\\//g\"     templates/*.twig templates/*/*.twig templates/*/*/*.twig templates/*/*/*/*.twig templates/*/*/*/*/*.twig",
+      "sed -I '' \"s/{% embed \"/{% embed \"@ts_motif\\//g\"   templates/*.twig templates/*/*.twig templates/*/*/*.twig templates/*/*/*/*.twig templates/*/*/*/*/*.twig",
+      "sed -I '' \"s/{{ function('wp_.*') }}//g\"              templates/*.twig templates/*/*.twig templates/*/*/*.twig templates/*/*/*/*.twig templates/*/*/*/*/*.twig",
+      "sed -I '' \"s/{{ function(\"wp_.*') }}//g\"             templates/*.twig templates/*/*.twig templates/*/*/*.twig templates/*/*/*/*.twig templates/*/*/*/*/*.twig",
+      "sed -I '' \"s/{% do action('.*') %}//g\"                templates/*.twig templates/*/*.twig templates/*/*/*.twig templates/*/*/*/*.twig templates/*/*/*/*/*.twig",
+      "sed -I '' \"s/{% do action('.*') %}//g\"                templates/*.twig templates/*/*.twig templates/*/*/*.twig templates/*/*/*/*.twig templates/*/*/*/*/*.twig",
+      "sed -I '' 's/featured_image: Image(308)//g'             templates/*.twig templates/*/*.twig templates/*/*/*.twig templates/*/*/*/*.twig templates/*/*/*/*/*.twig",
```

Comment:
`Image()` is deprecated in Timber 2, so if we need to kill this code, we should update it to the new syntax. It's in a PR in base-assets if it hasn't been merged already!